app/code/community/Algolia/Algoliasearch/Helper/Algoliahelper.php

```diff
@@ -84,6 +84,16 @@ public function mergeSettings($index_name, $settings)
         } catch (\Exception $e) {
         }
 
+        if (isset($settings['attributesToIndex'])) {
+            $settings['searchableAttributes'] = $settings['attributesToIndex'];
+            unset($settings['attributesToIndex']);
+        }
+
+        if (isset($onlineSettings['attributesToIndex'])) {
+            $onlineSettings['searchableAttributes'] = $onlineSettings['attributesToIndex'];
+            unset($onlineSettings['attributesToIndex']);
+        }
+
         $removes = array('slaves');
```

Comment:
Yes better, it would make sense to rename it when we set it as well (https://github.com/algolia/algoliasearch-magento/blob/037bc15692035dbbc9c4d4cb67705f4455238478/app/code/community/Algolia/Algoliasearch/Helper/Entity/Producthelper.php#L269), wdyt ?